src/main/java/com/ryszardpanda/Library/exceptions/BookExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+package com.ryszardpanda.Library.exceptions;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+@ControllerAdvice
+public class BookExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler({ResourceNotFoundException.class})
+    public ResponseEntity<Object> handleResourceNotFoundException(
+            Exception ex, WebRequest request){
+        return new ResponseEntity<Object>(
+                new ErrorMessage("Not Found"), new HttpHeaders(), HttpStatus.NOT_FOUND);
```

Comment:
mozesz dodac do swojej metody obslugujacej wyjatek ze przyjmuje nie Exception a przyjmuje ResourceNotFoundException -> dzieki temu jak bedziesz mial getter to mozesz pobracz w wyjatku status http

Reply:
```
return new ResponseEntity<Object>(
                new ErrorMessage(ex.getMessage()), new HttpHeaders(), ex.getHttpStatus());
```

Reply:
ResponseEntity.status(ex.getHttpStatus()).body(ex.getMessage())